### ISSUE TITLE
<Advanced HTML and CSS Course >: <Deal with reference to removed link/missing content>

### ISSUE THREAD

**Issue by Leo-U** [TheOdinProject/curriculum#25650]
### Describe your suggestion

In the past, an old link was removed. However, the lesson still mentions that link.

You can see where the old link used to be here under the Assignment section in this older lesson version: https://github.com/TheOdinProject/curriculum/blob/5f37d43908ef92499e95a9b90fc3cc291a95014c/html_css/accessibility/accessibility_auditing.md

The link that was removed is https://firefox-source-docs.mozilla.org/devtools-user/accessibility_inspector/index.html

In the current version, the reference to the old link is where the lesson says:
>Although there will be differences between the browsers, such as the value of the role property or how a11y properties are presented, also check out the “Features of the Accessibility panel” section mentioned below for MDN’s documentation. There is some useful information that, while more tailored to Firefox, can still be useful to a Chrome user.

As for the solution, presumably that would be either removing the reference to the link or putting the link back in.

Actually, now that I look at it, you can see that the number 1. is still included in the current version as well. Everything after 2. is no longer present for whatever reason. As you can see, it merely shows 1, while step 2. is missing entirely:

[screenshot](https://i.imgur.com/wIrGugI.png)

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-accessibility-auditing

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

**Comment by wise-king-sullyman** [comment]
Thanks for raising this issue!

Whoever takes it on should start with investigating if the removal of the Firefox content was intentional and just not complete or if it was a mistake (unless someone else on the maintainer team knows off hand) to determine if the appropriate course of action here is removing the reference or adding the content back.

**Comment by Eduardo06sp** [comment]
Same issue in the Rails path. 
https://www.theodinproject.com/lessons/advanced-html-and-css-accessibility-auditing#assignment

Given the statement:
> There is some useful information that, while more tailored to Firefox, can still be useful to a Chrome user.

It seems that the intention was to keep some of that content. Whether or not you would like to keep it I think is up to the maintainers.
-----------------
I would like to be assigned to this, unless you need a maintainer to take it!

**Comment by wise-king-sullyman** [comment]
@Eduardo06sp assigned, thanks!

Before making the changes I would look through the Git history and try to find the PR that removed the link and such. Hopefully there was a good description of what the reason for removing it was in that PR or an associated issue.

**Comment by Eduardo06sp** [comment]
@wise-king-sullyman From what I can tell, it looks like commit 1ffe735 was the time the change was made, from PR #23322 (which also links to the relevant, broader Issue).

I used git blame (on this & the file itself) to see if maybe the commit messages had more details not present on Github's GUI, but I did not find any deeper explanation other than just the message mentioning the removal of Firefox references & moving the file itself to a new location.

It seems to me that this was a simple oversight, as there were a decent number of files that had to be updated.

I would not want to make the decision as to what direction in to move in, so my thoughts are that we should find a Chromium-specific resource or just keep it. In Chrome's docs, I could not find an equivalent, brief description for the accessibility tree node properties, like there is on the page for Firefox (https://firefox-source-docs.mozilla.org/devtools-user/accessibility_inspector/index.html#features-of-the-accessibility-panel).

It can definitely still help to read through the link for Firefox, but the issue would be that users might get confused, since it's not the **exact** same thing they would be seeing in Chrome. A potential argument against this would be that given that this is already at Advanced HTML/CSS, users should have no issue understanding a slightly different resource (than they may expect) at this point, especially since it's still useful.

**Comment by wise-king-sullyman** [comment]
I agree with your assessment @Eduardo06sp, if we can't find an equivalent resource for Chrome it's probably better to re-add the Firefox resource.

**Comment by Eduardo06sp** [comment]
@wise-king-sullyman I was looking through articles & even some videos, and I could not find an equivalent resource that's Chromium-specific on my end.

There were indeed some good articles & videos online about Chrome's accessibility features, but they seem to gloss over what each property represents.

Am I okay to add the Firefox link back in for that part, or should I wait for other suggestions?

**Comment by wise-king-sullyman** [comment]
@Eduardo06sp yeah I would cosign on just adding the Firefox link.
